docs/operations/logging.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Tracing in SlateDB
+---
+
+# Tracing in SlateDB
+
+SlateDB uses the [`tracing`](https://github.com/tokio-rs/tracing) library for logging and diagnostics. This provides structured, contextual logging that helps debug and monitor your database operations.
+
+## Getting Started
+
+To enable logging in your SlateDB application, you'll need to add the following dependencies to your `Cargo.toml`:
+
+```toml
+[dependencies]
+tracing = "0.1"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+env_logger = "0.10"
+```
+
+## Basic Setup
+
+Here's a basic example showing how to initialize tracing with `env_logger` and SlateDB:
+
+```rust
+use slatedb::Database;
+use tracing::{info, warn, error};
+use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+
+#[tokio::main]
+async fn main() {
+    // Initialize tracing subscriber
+    tracing_subscriber::registry()
```

Comment:
We just do [`tracing_subscriber::fmt::init();`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/fn.init.html). I think this is roughly what you're doing.

Reply:
yea my bad, the example i put in here is not very relevant. I wanted to gather some inputs on the usage so I put a example as place holder. I will remove all the extra verbiage and update the example with a more relevant code snippet (will validate it using an actual test snippet that I can run).